lib/index.js

```diff
@@ -47,6 +48,20 @@ function register (server, options, next) {
     });
   };
 
+  var orgsEdit = function (id, orgObj, cb) {
+    return query(queries.orgsEdit(id, orgObj), pool,  function (err, res) { //eslint-disable-line
+      if (err) {
+        return cb(err);
+      }
+      // if there were no rows affected, the organisation id does not exist.
+      if (res.rowCount === 0) {
+        return cb(Boom.notFound('That organisation does not exist'));
```

Comment:
@SimonLab http://bdadam.com/blog/video-douglas-crockford-about-the-new-good-parts.html
9mins 20secs